src/seedu/addressbook/data/person/UniquePersonList.java

```diff
@@ -1,15 +1,10 @@
 package seedu.addressbook.data.person;
 
-import java.util.ArrayList;
-import java.util.Arrays;
-import java.util.Collection;
-import java.util.Collections;
-import java.util.Iterator;
-import java.util.List;
-
 import seedu.addressbook.common.Utils;
 import seedu.addressbook.data.exception.DuplicateDataException;
 
+import java.util.*;
```

Comment:
Coding standard does not allow wildcard imports. Import the specific class explicitly.